src/commands/Minion/favorite.ts

```diff
@@ -19,6 +19,24 @@ export default class extends BotCommand {
 	async run(msg: KlasaMessage, [items]: [Item[] | undefined]) {
 		const currentFavorites = msg.author.settings.get(UserSettings.FavoriteItems);
 
+		if (msg.flagArgs.clear) {
+			const currentFavorites = msg.author.settings.get(UserSettings.FavoriteItems);
+			if (currentFavorites.length > 0) {
+				await msg.confirm(
+					`Do you want to clear your favorite items from your bank? You currently have ${currentFavorites.length.toLocaleString()} favorite items.`
```

Comment:
They're not clearing it from 'their bank'.


`Are you sure you want to clear your favorite items list?`